lab-michael/test/routes-test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const expect = require('chai').expect;
+const request = require('superagent');
+
+require('../server.js');
+
+
+describe('Testing the recipe routes', function() {
+  // let recipe = null;
+  describe('Testing Invalid Routes', function() {
+    it('should return error bad route', function(done) {
+      request.get('localhost:3000/api/recipeisee')
+      .end((err, res) => {
+        expect(res.status).to.equal(404);
+        done();
+      });
+    });
+  });
+  describe('Testing Valid Routes', function() {
+    it('should call the root directory /api/recipe', function(done) {
+      request.get('localhost:3000/api/recipe')
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        done();
+      });
+    });
+  });
+  describe('GET TEST /api/recipe', function() {
+    it('should show the recipe by ID 4f04073e-48fe-42e0-a440-03111ace855d', function(done) {
+      request.get('localhost:3000/api/recipe/4f04073e-48fe-42e0-a440-03111ace855d')
```

Comment:
Rather than testing data that exists, create/destroy resources using your before and after blocks.